### PR TITLE
Author show page #14

### LINKED ISSUE #14: User Story 14, Author Show Page
```
As a Visitor,
When I visit an author's show page
I see all books by that author
Each book should show:
- the book title
- the number of pages in the book
- the year the book was published
- a small image of the book cover

Each book should show a list of any other authors
(exclude this show page's author from that list)
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #15

child of #33

depends on #17

### PR DESCRIPTION
**Pull request by Vjp888**
Adds routing and show page

adds, title, page count, publishing date, thumbnail and co-authors to show page.
adds instance method to book model to remove a given author.

ensures that titles are links to a book show page 
ensures that co-authors are links to author show pages

hides co-authors if there are none present

all tests passing.

closes #14

### CODE REVIEW

File: spec/features/author_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    within "#book-#{book_1.id}" do
+      expect(page).to have_xpath("//img[@src='steve.jpg']")
+      expect(page).to have_content("Title: book title 1")
+      expect(page).to have_content("Page Count: #{book_1.pages}")
+      expect(page).to have_content("Year Published: #{book_1.year_published}")
+      expect(page).to_not have_content("Bob")
+    end
+
+    within "#book-#{book_2.id}" do
+      expect(page).to have_xpath("//img[@src='steve.jpg']")
```
Comment (milevy1) [inline spec/features/author_show_spec.rb]: This should expect img to be 'andrew.jpg' . Or you could change these to just string interpolating as  #{book_2.thumbnail}

Why is it passing the spec currently?

**Comment by milevy1** [comment]
Looks good Vince.  Just one comment on the spec/features/author_show_spec.rb.  Then I approve merging.
